code/modules/vending/vending.dm

```diff
@@ -1492,19 +1492,19 @@
 
 	create_products()
 		..()
-		product_list += new/datum/data/vending_product(/obj/item/device/pda2, 10, cost=100)
-		product_list += new/datum/data/vending_product(/obj/item/disk/data/cartridge/atmos, 2, cost=40)
-		product_list += new/datum/data/vending_product(/obj/item/disk/data/cartridge/mechanic, 2, cost=40)
-		product_list += new/datum/data/vending_product(/obj/item/disk/data/cartridge/quartermaster, 2, cost=50)
-		product_list += new/datum/data/vending_product(/obj/item/disk/data/cartridge/medical, 2, cost=40)
-		product_list += new/datum/data/vending_product(/obj/item/disk/data/cartridge/genetics, 2, cost=40)
-		product_list += new/datum/data/vending_product(/obj/item/disk/data/cartridge/toxins, 2, cost=50)
-		product_list += new/datum/data/vending_product(/obj/item/disk/data/cartridge/botanist, 2, cost=40)
-		product_list += new/datum/data/vending_product(/obj/item/disk/data/cartridge/janitor, 2, cost=40)
-		product_list += new/datum/data/vending_product(/obj/item/disk/data/cartridge/engineer, 2, cost=70)
+		product_list += new/datum/data/vending_product(/obj/item/device/pda2, 20, cost=100)
+		product_list += new/datum/data/vending_product(/obj/item/disk/data/cartridge/atmos, 5, cost=40)
+		product_list += new/datum/data/vending_product(/obj/item/disk/data/cartridge/mechanic, 5, cost=40)
```

Comment:
The analysis made easy cartridge is just a cheaper and slightly better version of the networking cartridge. I recommend dumping it from this vendor. If you do, I'll merge this.